SmartContractSample/Program.cs

```diff
@@ -48,7 +48,10 @@ private static async Task DeployContract(IClient client)
 
             // Create entry
             var sources = new[] { File.ReadAllText("sc\\sc1.cs") };
-            var dependencies = new[] { "Miyabi.Binary.Models", "Miyabi.ModelSdk"};
+            var dependencies = new[] { 
```

Comment:
Aren't these part of the Internal White list?
```
            "Miyabi.NodeSdk",
            "Miyabi.Asset.Models",
            "Miyabi.Binary.Models",
            "Miyabi.Common",
            "Miyabi.Contract.Models",
            "Miyabi.Contract.Dependencies",
            "Miyabi.ContractSdk",
            "Miyabi.Cryptography.Secp256k1",
            "Miyabi.Entity.Models",
            "Miyabi.ModelSdk",
            "Miyabi.NFT.Models",
            "Miyabi.PrivateData.Models",
```

Reply:
Ok, I will remove unnesessary dependencies from this sample.